inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -0,0 +1,223 @@
+"""Bathymetry class and utilities."""
+
+import random
+from typing import Optional, Sequence, Tuple, Union
+
+import matplotlib
```

Comment:
why not do `from matplotlib.pyplot as plt `, does it go against the google style guide?

Reply:
I think either `import matplotlib` or `import matplotlib.pyplot as plt` are according to the Google Python style guide. In this file in particular I need to access `matplotlib` (to get the axes class, `matplotlib.axes.Axes`) and `matplotlib.pyplot` (to create and close the figure), so I decided to go with one import only.

Reply:
Makes sense :)